sdk/keyvault/Azure.Security.KeyVault.Administration/tests/KeyVaultSettingsClientLiveTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Security.KeyVault.Administration.Tests
+{
+    public class KeyVaultSettingsClientLiveTests : SettingsTestBase
```

Comment:
Do we need to test the other operations as well?

Reply:
They are in the sample that's also a recorded test. Didn't figure we need to test 2/3<sup>rds</sup> of the operations twice. Do you feel we should for some other reason?

Reply:
Or was it just that it wasn't obvious? I could add a comment pointing to the sample if that would help. KV tests already take long enough, I didn't want to compound it further unnecessarily. Sure, this is quick, but it all adds up.

Reply:
I added a comment. Let me know if you want more, though.